app/handlers/objectTracking.py

```diff
@@ -17,5 +27,56 @@ class ObjectTrackingHandler(tornado.web.RequestHandler):
 
     @apiError error_message The error message to display.
     """
+
     def post(self):
+        self.objectTrack(self.request.body_arguments["identifier"])
+        
+        message = "Hello,\n\tWe have finished processing your video and identifying all objects.\nThank you for your patience,\nThe Santos Team"
+        subject = "Your video has finished processing."
+
+        EmailHelper.send_email(self.request.body_arguments["email"], subject, message)
+
         self.finish("Object Tracking")
+
+
+    def objectTrack(self, identifier):
+        """
+        Runs TrafficIntelligence trackers and support scripts.
+        """
+        project_path = get_project_path(identifier)
+
+        # create test folder
+        if not os.path.exists(os.path.join(project_path, "run")):
+            os.mkdir(os.path.join(project_path, "run"))
+
+        tracking_path = os.path.join(project_path, "run", "run_tracking.cfg")
+
+        # removes object tracking.cfg
+        if os.path.exists(tracking_path):
+            os.remove(tracking_path)
+
+        # creates new config file
+        prev_tracking_path = os.path.join(project_path, ".temp", "test", "test_object", "object_tracking.cfg")
+        shutil.copyfile(prev_tracking_path, tracking_path)
+
+        update_dict = {'frame1': 0, 
+            'nframes': 0, 
+            'database-filename': 'results.sqlite', 
+            'classifier-filename': os.path.join(project_path, "classifier.cfg"),
+            'video-filename': get_project_video_path(identifier),
+            'homography-filename': os.path.join(project_path, "homography", "homography.txt")}
+        update_config_without_sections(tracking_path, update_dict)
+
+        db_path = os.path.join(project_path, "run", "results.sqlite")
+
+        if os.path.exists(db_path):  # If results database already exists,
+            os.remove(db_path)  # then remove it--it'll be recreated.
+        subprocess.call(["feature-based-tracking", tracking_path, "--tf", "--database-filename", db_path])
+        subprocess.call(["feature-based-tracking", tracking_path, "--gf", "--database-filename", db_path])
+
+        subprocess.call(["classify-objects.py", "--cfg", tracking_path, "-d", db_path])  # Classify road users
+
+        db_make_objtraj(db_path)  # Make our object_trajectories db table
+
+        video.create_tracking_video(project_path, get_project_video_path(identifier))
```

Comment:
We might want to remove this for sake of time, but if we want it to check object tracking for now, then keep it.